huaweicloud/services/acceptance/cc/resource_huaweicloud_cc_global_connection_bandwidth_associate_test.go

```diff
@@ -0,0 +1,319 @@
+package cc
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+
+	"github.com/chnsz/golangsdk"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/services/acceptance"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/utils"
+)
+
+func getGCBAssociateFunc(cfg *config.Config, state *terraform.ResourceState) (interface{}, error) {
+	client, err := cfg.NewServiceClient("cc", acceptance.HW_REGION_NAME)
+	if err != nil {
+		return nil, fmt.Errorf("error creating CC client: %s", err)
+	}
+	getGCBHttpUrl := "v3/{domain_id}/gcb/gcbandwidths/{id}"
+	getGCBPath := client.Endpoint + getGCBHttpUrl
+	getGCBPath = strings.ReplaceAll(getGCBPath, "{domain_id}", cfg.DomainID)
+	getGCBPath = strings.ReplaceAll(getGCBPath, "{id}", state.Primary.ID)
+
+	getGCBOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+	}
+
+	getGCBResp, err := client.Request("GET", getGCBPath, &getGCBOpt)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving global connection bandwidth: %s", err)
+	}
+
+	getGCBRespBody, err := utils.FlattenResponse(getGCBResp)
+	if err != nil {
+		return nil, fmt.Errorf("error retrieving global connection bandwidth: %s", err)
+	}
+
+	instances := utils.PathSearch("globalconnection_bandwidth.instances", getGCBRespBody, make([]interface{}, 0))
+	if v, ok := instances.([]interface{}); ok && len(v) == 0 {
+		return nil, golangsdk.ErrDefault404{}
+	}
+	return instances, nil
+}
+
+func TestAccGCBAssociate_basic(t *testing.T) {
+	var obj interface{}
+
+	name := acceptance.RandomAccResourceName()
+	rName := "huaweicloud_cc_global_connection_bandwidth_associate.test"
+
+	rc := acceptance.InitResourceCheck(
+		rName,
+		&obj,
+		getGCBAssociateFunc,
+	)
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck: func() {
+			acceptance.TestAccPreCheck(t)
+			acceptance.TestAccPreCheckEpsID(t)
```

Comment:
looks it's not necessary to use eps ID in this test

Reply:
fixed.